asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/process/wps/WPSProcess.kt

```diff
@@ -1,19 +1,23 @@
 package org.n.riesgos.asyncwrapper.process.wps
 
+import org.n.riesgos.asyncwrapper.config.RetryConfiguration
 import org.n.riesgos.asyncwrapper.config.WPSOutputDefinition
 import org.n.riesgos.asyncwrapper.process.*
 import org.n.riesgos.asyncwrapper.process.Process
 import org.n52.geoprocessing.wps.client.ExecuteRequestBuilder
+import org.n52.geoprocessing.wps.client.WPSClientException
 import org.n52.geoprocessing.wps.client.WPSClientSession
 import org.n52.geoprocessing.wps.client.encoder.WPS20ExecuteEncoder
 import org.n52.geoprocessing.wps.client.model.*
 import org.n52.geoprocessing.wps.client.model.execution.Data
-import java.sql.Ref
 import java.util.*
 import java.util.logging.Logger
 
 
-class WPSProcess(private val wpsClient : WPSClientSession, private val url: String, private val processID: String, private val wpsVersion: String, private val expectedOutputs : List<WPSOutputDefinition>) : Process {
+class WPSProcess(private val wpsClient : WPSClientSession, private val url: String, private val processID: String, private val wpsVersion: String, private val expectedOutputs : List<WPSOutputDefinition>, private val retryConfig : RetryConfiguration) : Process {
+
+    private var retryAttempts = 0;
+    constructor(wpsClient : WPSClientSession, url: String, processID: String, wpsVersion: String, expectedOutputs : List<WPSOutputDefinition>) : this(wpsClient, url, processID, wpsVersion, expectedOutputs, RetryConfiguration(3, 3000))
```

Comment:
With the overall config in the main constructor, do we still need this side constructor?

Reply:
Still a question: Do we use this secondary constructor? If we need it, then we don't give the arguments from the application.yml - which is not what we want.

I would like you to remove that secondary constructor.